src/models/transfers.js

```diff
@@ -29,6 +29,8 @@ const Condition = require('five-bells-condition').Condition
 
 const validExecutionStates = ['prepared']
 const validCancellationStates = ['prepared', 'proposed']
+const SIGNATURE_TYPE_ED25519 = 'ed25519-sha512'
+const SIGNATURE_TYPE_SHA256 = 'sha256'
```

Comment:
I discussed with @justmoon and we were thinking that it might be more accurate to call these "commitments" rather than "signatures". I'm still not fully caught up with understanding how this implementation works, but does this sound right? https://en.wikipedia.org/wiki/Commitment_scheme

It seems like we aren't using it in the typical way that commitments are used, but cryptographically it seems to be closer to a commitment than a signature.